core/src/main/java/org/opensearch/sql/data/model/ExprBooleanValue.java

```diff
@@ -19,7 +19,7 @@
 
   private final Boolean value;
 
-  private ExprBooleanValue(Boolean value) {
+  public ExprBooleanValue(Boolean value) {
```

Comment:
why change to public?

Reply:
Need to reference it in `resultBuilder.put(resultKeyName, new ExprBooleanValue(columnValue.booleanValue()));`

Reply:
use ExprBooleanValue.of() method instead.